gnucash/gtkbuilder/dialog-price.glade

```diff
@@ -9,19 +9,19 @@
     </columns>
     <data>
       <row>
-        <col id="0" translatable="yes">Bid</col>
+        <col id="0" translatable="yes" context="In commoditiy quote dialog">Bid</col>
       </row>
       <row>
-        <col id="0" translatable="yes">Ask</col>
+        <col id="0" translatable="yes" context="In commoditiy quote dialog">Ask</col>
       </row>
       <row>
-        <col id="0" translatable="yes">Last</col>
+        <col id="0" translatable="yes" context="In commoditiy quote dialog">Last</col>
       </row>
       <row>
-        <col id="0" translatable="yes">Net Asset Value</col>
+        <col id="0" translatable="yes" context="In commoditiy quote dialog">Net Asset Value</col>
       </row>
       <row>
-        <col id="0" translatable="yes">Unknown</col>
+        <col id="0" translatable="yes" context="In commoditiy quote dialog">Unknown</col>
```

Comment:
"In commodity quote dialog" isn't terribly helpful, the translator still has to go figure out what that dialog is (In spite of @fellen's protestation it's actually called the **Price Editor** dialog. Also in spite of his protestation `Bid` and `Ask` are types of quotes--bid if you want to sell, ask if you want to buy--but `Last` and `Net Asset Value` are prices--the former the price of the last trade on an exchange prior to the time of the price, the latter the price of an open-end mutual fund calculated after the market close each day and for which both buys and sells of that fund from the fund company are priced the following trading day.

That explanation is a bit wordy for translation contexts, so maybe `Commodity quote type` for `Bid` and `Ask`, `Commodity price type` for `Last` and `Unknown`, and `Open ended mutual fund price` for `NetAsset Value`.

Reply:
Yes, this context is totally redundant. The location[s] is/are already in the file before the msgid. This info is also displayed in weblate's interface in the lower right corner.

@jralls My definition

- Price: amount they user self (got) paid per piece.
- Quote: Offers the user got aware of. Often functions (Open/Close, Hi/Lo, (current) Ask/Bid …) of the sets of prizes paid by others. Also the NAV is the offer of the emitter to buy the funds back on the next working day.

But I agree with @brianhsu that the english language lacks precission in many use cases. ;-)

Can we delay this thread behind the next release?

Reply:
@fellen, in English "quote" is actually short for "quote a price", where "quote" is a somewhat antique usage for "speak". It's not used for prices of transactions in the past, regardless of who transacted them.

> My definition

"When I use a word," [Humpty Dumpty](https://en.wikipedia.org/wiki/Humpty_Dumpty) said, in rather a scornful tone, "it means just what I choose it to mean—neither more nor less."

> Can we delay this thread behind the next release?

Yes.

Reply:
> Yes, this context is totally redundant. The location[s] is/are already in the file before the msgid. This info is also displayed in weblate's interface in the lower right corner.
> 
> @jralls My definition
> 
>     * Price: amount they user self (got) paid per piece.
> 
>     * Quote: Offers the user got aware of. Often functions (Open/Close, Hi/Lo, (current) Ask/Bid …) of the sets of prizes paid by others. Also the NAV is the offer of the emitter to buy the funds back on the next working day.
> 
> 
> But I agree with @brianhsu that the english language lacks precission in many use cases. ;-)
> 
> Can we delay this thread behind the next release?

I totally agree it's redundant in current code base.  This PR is only useful in the older version, because some report use "Last" in their option drop-down menu to indicate time instead of type of price. And they should translate to quite different word in Chinese. (`成交價` vs `最後的`)

Fortunately, the latest code base does not have this issue, because "Ask" / "Last" does not been used in other place. 

Also if the "Rate" column inside gnc-tree-view-split-reg.c  / split-register-model.c is a hidden column and should never displayed to users, then I have no problem that it remain current status and has no context. 

Actually after all of these discussion and clarification,  I don't think this PR is actually needed and totally fine if this PR get closed without further action . Because all of these word that actually displayed to users does not have ambiguity in Chinese anymore.

The only two thing I'm not sure is that:

1. What the "Rate" in the job dialog actually means. Does it mean tax rate or exchange rate or interest rate or something else? This really impact the quality of Chinese translation as I mentioned before. And I couldn't find explanation of this field in the GnuCash manual on the website or the help documentation in the program.
2. Does the "Rate" in the gnc-tree-view-split-reg.c  / split-register-model.c refer to the account register column? I think it is but I'm not sure because I actually have a account register that display this column as the following screenshot, but according to above discussion it should be a hidden column. If this is just a bug and user should never see this column, I'm totally agree that "Rate" don't need any context.

![Screenshot](https://imgur.com/GmXdH3u.png)

Reply:
> Fortunately, the latest code base does not have this issue, because "Ask" / "Last" does not been used in other place.

It doesn't matter, it still requires the translator to go trolling around in the code and hope to be able to figure out the meaning. Like I said earlier, *all* single-word msgids should have context so that translators don't need to do that.

> What the "Rate" in the job dialog actually means?

That sadly took a bit of digging to find out. It's from a [drive-by contribution](https://bugs.gnucash.org/show_bug.cgi?id=728136) 7 years ago and isn't documented anywhere. The answer is that it's the unit price of whatever the job is associated with: It could be an hourly or a piece rate for labor or a material unit price.

> Does the "Rate" in the gnc-tree-view-split-reg.c / split-register-model.c refer to the account register column? 

No. It's the same as the price column in a non-currency register, that is `value / amount`. It's used for convenience by the balancing code and is set by the Transfer Dialog's exchange rate section.